materialize-databricks/driver.go

```diff
@@ -277,6 +279,12 @@ func (c client) PreReqs(ctx context.Context, ep *sql.Endpoint) *sql.PrereqErr {
 		case databricksSql.StateStopping:
 			errs.Err(fmt.Errorf("The selected SQL Warehouse is stopping, please start the SQL warehouse and try again."))
 		}
+
+		if res.AutoStopMins >= 15 {
```

Comment:
Uber-nit: We probably don't need to log this if the materialization is already explicitly configured to have a 0s update delay